src/test/java/com/salesforce/dynamodbv2/testsupport/ArgumentBuilder.java

```diff
@@ -81,14 +84,14 @@
             .withBinaryHashKey(true)
             .build()),
 
-        HashPartitioning(root -> SharedTableBuilder.builder()
+        HashPartitioning(root -> getTrivialCompositeClient(SharedTableBuilder.builder()
```

Comment:
In the interest of not adding significantly to the test runtime, I'm making the hash partitioning test argument a composite client of one -- so we're testing both here.